### ISSUE TITLE
Document suggested package naming convention

### ISSUE THREAD

**Issue by timkelty** [suitcss/suit#112]
e.g. suitcss-components-*, suitcss-utils-*

**Comment by necolas** [comment]
is anyone writing open source components and utils? should they be?

**Comment by necolas** [comment]
oh i just saw https://github.com/suitcss/suit/issues/110, ok that's cool.

**Comment by timkelty** [comment]
I have some I just need to publish.
I know https://github.com/trunkclub has a few as well.

**Comment by timkelty** [comment]
https://github.com/trunkclub/suitcss-components-form
https://github.com/trunkclub/suitcss-components-table

**Comment by giuseppeg** [comment]
A good first step would be to write down some guidelines and requirements for components to qualify.
The big question there is how do we keep them consistent? @necolas do you guys have global variables for margin, padding, font-size etc?

**Comment by timkelty** [comment]
It seems a component should always stand alone, so to me, consistency between component's variables isn't that important.

I usually end up something like:

```
:root {
  --App-gutter-size: 20px;
}
...
:root {
  --Arrange-gutter-size: var(--App-gutter-size);
  --Grid-gutter-size: var(--App-gutter-size);
}
```

**Comment by oleersoy** [comment]
One possible requirement is that suit components should be [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself) by assuming the presence of the `normalize.css / base.css ` ([Base Styles](https://github.com/suitcss/base)) layer.

Designers may want to leave `suit` off the name of the component for the reason that the component works across various frameworks, and respects the structure and conventions of all of them.

I would be great if the theme styling methodology worked really well across autonomous components...maybe it does...Need to experiment more with it.  How do we get autonomous components to employ variables so that consistency across them can be plugged (Color palette, font, use of gradients, drop shadows, animation timing, etc)?

**Comment by giuseppeg** [comment]
@timkelty yeah :+1:  we should make that clear and a requirement I guess :)
